issues.go

```diff
@@ -332,6 +332,7 @@ type ListProjectIssuesOptions struct {
 	UpdatedAfter       *time.Time `url:"updated_after,omitempty" json:"updated_after,omitempty"`
 	UpdatedBefore      *time.Time `url:"updated_before,omitempty" json:"updated_before,omitempty"`
 	Confidential       *bool      `url:"confidential,omitempty" json:"confidential,omitempty"`
+	DueDate            *string    `url:"due_date,omitempty" json:"due_date,omitempty"`
```

Comment:
Please place this between `CreatedBefore` and `UpdatedAfter` following the ordering in the docs... Thanks!